src/test/regress/expected/create_drop_database_propagation.out

```diff
@@ -932,6 +928,259 @@ SELECT * FROM public.check_database_on_all_nodes('test_db') ORDER BY node_type;
 REVOKE CONNECT ON DATABASE test_db FROM propagated_role;
 DROP DATABASE test_db;
 DROP ROLE propagated_role, non_propagated_role;
+-- test create / drop database commands from workers
+-- remove one of the workers to test node activation too
+SELECT 1 from citus_remove_node('localhost', :worker_2_port);
+ ?column?
+---------------------------------------------------------------------
+        1
+(1 row)
+
+\c - - - :worker_1_port
+CREATE DATABASE local_worker_db;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to other nodes
+HINT:  You can manually create a database and its extensions on other nodes.
```

Comment:
Sidenote: Should we update this hint message to include a hint about citus.enable_create_database_propagation? In any case this would be better to do in a separate PR.

Reply:
makes sense, let me update this in a separate PR